torch/_dynamo/compile_metrics.py

```diff
@@ -0,0 +1,9 @@
+# Global information for the purposes of adding to CompilationMetrics
+
+# From Inductor's reinplacing pass: the number of Tensors that we failed to reinplace.
+possibly_missed_reinplacing_opportunities: int = 0
```

Comment:
Can we just do this with `counters` from `torch/_dynamo/utils.py` ? applying deltas on cache hit is already supported for fx graph cache hit with counters

Reply:
Good point, I'll just use counters